ci/updatePackageForRelease.js

```diff
@@ -0,0 +1,99 @@
+const fs = require('fs');
+
+const NPM_PACKAGE_PATH = `${__dirname}/../src/electron/npm`;
+const PACKAGE_JSON_PATH = `${NPM_PACKAGE_PATH}/package.json`;
+const ELECTRON_D_TS_PATH = `${NPM_PACKAGE_PATH}/electron.d.ts`;
+
+module.exports = async ({github}) => {
+  const {VERSION} = process.env;
+  // Get the release by version. This will give us the asset ids
+
+  // TODO: Download electron.d.ts from release and move into npm
```

Comment:
It looks like we do this on lines 26-34 below, if that's the case, we can probably remove the TODO

Reply:
removed todo line